classes/submission/reviewAssignment/ReviewAssignment.php

```diff
@@ -899,9 +869,10 @@ class_alias('\PKP\submission\reviewAssignment\ReviewAssignment', '\ReviewAssignm
         'SUBMISSION_REVIEW_METHOD_ANONYMOUS',
         'SUBMISSION_REVIEW_METHOD_DOUBLEANONYMOUS',
         'SUBMISSION_REVIEW_METHOD_OPEN',
-        'REVIEW_ASSIGNMENT_NOT_UNCONSIDERED',
+        'REVIEW_ASSIGNMENT_NEW',
```

Comment:
These are only here for legacy code and we'll remove them at some point. Since old code won't refer to the new constants, I'd suggest removing them from this list.

Reply:
Removed .